### PR TITLE
initial DELETE stock route [SB-1]

### PR DESCRIPTION
**Pull request by SameedB**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
New Feature to the program

* **What is the current behavior?What does the code do before you made the new changes?** 
No delete option 

* **What is the new behavior?**
allows for users to delete a stock added to the DB using a userId and stockId

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
no, should be running normally

* **Other information**:

### CODE REVIEW

**Review by arsoljon — CHANGES_REQUESTED** [review]
Github doesn't let me select the lines for the code I want to comment on because the most recent commit includes merging main into this branch. 

Line 35, the url could be shortened to exclude '/stocks'. 
Line 36 and 37 use the params method incorrectly but in the case of this delete function, I think it would be best to use the .query method. I did some reading and found a link explaining why. https://coursework.vschool.io/express-params-and-query/

**Review by jorgecan101 — DISMISSED** [review]
Looks good to me

File: index.js [hunk 1]
```diff
@@ -1,46 +1,66 @@
...
+        })
+        .catch(function(error){
+            res.json({"message":"Unsuccessful"})
+
+        })
+})
+
+
+app.delete('/deleteStock', (postmanRequest, postmanResponse) => {
+    const { userId } = postmanRequest.query.userId
```
Comment (arsoljon) [inline index.js]: When I ran a test, line 36 and 37 returned empty when I printed them and I received a "{ msg: 'User not found' }" when successfully running the delete request. I don't think the variables are working with braces around them. Consider removing the braces.

**Review by arsoljon — COMMENTED** [review]
Everything is looking good. 1 more change is needed.

**Review by arsoljon — APPROVED** [review]
Looks good to me!

**Review by jorgecan101 — APPROVED** [review]
LGTM

**Pull request merged**